### ISSUE TITLE
关于CORAL loss

### ISSUE THREAD

**Issue by yyssmm** [jindongwang/transferlearning#187]
王老师：您好
请问您DDC_DeepCoral里面的关于CORAL loss的代码，
    # frobenius norm
    loss = (cs - ct).pow(2).sum().sqrt()
    loss = loss / (4 * d * d)
请问这里之不是多了一个开方（sqrt）啊？谢谢

**Comment by jindongwang** [comment]
这个代码是有点问题。因为coral在这个仓库里 有好几个新旧版本，可能搞乱了。正确的coral代码你可以看这个：https://github.com/jindongwang/transferlearning/blob/master/code/distance/coral_pytorch.py
